.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,15 @@ Please label this PR with one of the following labels, depending on the scope of
 - Docs
 -->
 
+## What is the problem this PR solves?
+
+// Please do not just reference an issue
+
+## How does this PR solve the problem?
+
+// Explain how you solved the problem in your code.
+
+
 ## What does this PR do?
```

Comment:
I think the 2 new sections, which I like would satisfy this section. What would we add to this section that would not probably be already explained in the previous 2 sections?

Reply:
Agree, will remove it too.